enhancements/etcd/automated-backups-no-config.md

```diff
@@ -0,0 +1,346 @@
+Upon creating a `Backup` CR on an Openshift cluster, the `PeriodicBackupController` reacts by creating a DaemonSet resource, with the `etcd-backup-server` container and the configurations applied in the CR.
+During [testing](https://github.com/openshift/cluster-etcd-operator/pull/1354#issuecomment-2408661261), this method has shown a solid and stable deployment, where the Backup pods shown only `1` restart over a duration of `45` minutes. Moreover, the cause of the restart on all the Backup pods were due to `node` issue, not the backup container.
+
+_**Pros**_
+
+As the Backup pods are being scheduled independently of the etcd pods, any error within the Backup pods does not influence cluster Openshift cluster availability.
+
+Moreover, as the Backup pods are being scheduled within every master nodes, the backups are being taken from every local etcd member on the same node. This avoids overwhelming a specific member with backup requests.
+
+Since a Backup pod exist on every master node, and a `hostPath` volume is being used to store the backups, it is guaranteed to have available backups on all master nodes, which is required for disaster recovery scenarios.
+
+_**Cons**_
+
+No issue has been encountered during testing this approach.
+
+### Open Questions
+
+- What values could be used for default configurations, considering all OCP variants.
+  - This has been discussed over the reviews.
+  - The default schedule to run once at midnight, which is expressed as `0 0 * * *`.
+  - The default retention is to keep the backups over the last `72h` (i.e. the last three backups).
+- How to handle an empty `schedule` ?
+  - This has been fixed, by using a default schedule to run the backup once at midnight.
+  - This configuration could be overridden by user supplied schedule.
+- How to mitigate losing backups upon control-plane scaling using `CPMSO` ?
+  - This has been fixed by using a label to identify the nodes with `voting` etcd member, and using the label during scheduling the backup pods to master nodes.
+- What is the minimum disk size needed in order to operate this feature without running out of disk space ?
+
+### Implementation Details/Notes/Constraints
+
+The _no-config_ backups builds atop of the existing [automated backup of etcd](https://docs.openshift.com/container-platform/4.15/backup_and_restore/control_plane_backup_and_restore/backing-up-etcd.html#creating-automated-etcd-backups_backup-etcd).
+
+The `PeriodicBackupController` within **cluster-etcd-operator** checks for `Backup` CR with `name=default` within every sync. Upon applying a `Backup` CR with `name=default`, the _no-config_ backup sidecar container is being enabled within each etcd pod in Openshift cluster.
+The `Backup` configurations such as `schedule`, `timezone` and `retention policy`, are being supplied to the _no-config_ container as flags. 
```

Comment:
So how does retention actually work for the hostpath based backup in the DS approach?